api/v1alpha1/bgppeer_types.go

```diff
@@ -65,6 +65,10 @@ type BGPPeerSpec struct {
 	// +optional
 	HoldTime time.Duration `json:"holdTime,omitempty" yaml:"hold-time,omitempty"`
 
+	// Requested BGP keepalive time, per RFC4271.
+	// +optional
+	KeepaliveTime time.Duration `json:"keepaliveTime,omitempty" yaml:"keepalive-time,omitempty"`
```

Comment:
can you add min / max thresholds? (also to holdtime)

Reply:
min/max are only allowed for intg not for time.Duration type